server/sockets/list.js

```diff
@@ -284,16 +262,13 @@ const clearVote = (io, listClients, viewersIds) => data => {
 };
 
 const updateList = (io, dashboardViewClients, cohortViewClients) => data => {
-  const { listId } = data;
+  const { listId, doc: list, ...rest } = data;
 
-  io.sockets.to(listChannel(listId)).emit(ListActionTypes.UPDATE_SUCCESS, data);
+  io.sockets
+    .to(listChannel(listId))
+    .emit(ListActionTypes.UPDATE_SUCCESS, { listId, ...rest });
 
-  updateListOnDashboardAndCohortView(
-    io.sockets,
-    listId,
-    dashboardViewClients,
-    cohortViewClients
-  );
+  emitListsMetaData(io)(cohortViewClients, dashboardViewClients)(list);
```

Comment:
I would go for `updateListOnDashboardAndCohortView`  name.
Application is getting bigger and this name is more descriptive.